internal/testrunner/runners/pipeline/test_result.go

```diff
@@ -16,7 +16,14 @@ import (
 
 const expectedTestResultSuffix = "-expected.json"
 
-var errTestCaseFailed = errors.New("test case failed")
+type ErrTestCaseFailed struct {
+	Reason  string
+	Details string
+}
+
+func (e ErrTestCaseFailed) Error() string {
```

Comment:
Doesn't linter complain about missing comment?

Reply:
Hmmm, strangely it doesn't 🤔 . Anyway I'll add a godoc comment here. Thanks for catching!

Reply:
Added in af2dbf6.